### ISSUE TITLE
Logo no longer appearing

### ISSUE THREAD

**Issue by NathanDiMauro** [lcdi/snipe-it#12]
Our logo image no longer appears on the hompage or the login screen. This seems to have broken when we fixed the "D'oh documentRoot" error #11. @lenora4321 has suggested it could be something with the config files.

**Comment by NathanDiMauro** [comment]
This [issue](https://github.com/snipe/snipe-it/issues/8766) from main Snipe-it seems to be the same as ours. 
https://github.com/snipe/snipe-it/issues/8766

**Comment by lenora4321** [comment]
Fixed!  I changed two things: first, I edited a line in the /etc/httpd/conf.d/inventory.lcdi.conf where it said "redirect" from "https://" to "http://".
Then, I changed the .env file in the live directory at /var/www/snipe-it on the line for ```APP_URL``` from ```inventory.lcdi``` to ```http://inventory.lcdi```.  (Nate pointed out that this might be an issue, so thanks!).

Now, we just need a new image, because the one up there now has an odd white boarder that's not pretty.  This is just the image itself (I uploaded a new one at some point in testing, and it apparently was not the best).  
![image](https://user-images.githubusercontent.com/54592240/112360741-04a1b400-8ca9-11eb-833c-f7494feef4ab.png)
